### DISCUSSION TITLE
@trpc/nuxt-query?

### DISCUSSION THREAD

**Discussion by kylegl** [wobsoriano/trpc-nuxt#100] [Ideas]
Hey man love this package. Quick question: have you ever thought about integrating this with [TanStack Query](https://tanstack.com/query/v4)? 

TRPC has [@trpc/react-query](https://trpc.io/docs/react-query) would be awesome to have an @trpc/nuxt(vue)-query. I'm sure your busy, just wondering if you seen/thought about this and if you have any pointers on how to get this started?

Thanks for all your hard work on trpc-nuxt!

**Comment by wobsoriano** [comment]
Might probably do it in a separate package. Let's see!

**Comment by ozum** [comment]
@wobsoriano and @kylegl, please have a look at this PR: #65.

**Comment by warflash** [comment]
Hey, just checking in here real quick. Is this something that is being considered/on the horizon? Would be a very valuable and welcome addition I think 😊
